package.json

```diff
@@ -15,20 +22,40 @@
     "type": "git",
     "url": "http://github.com/chriso/validator.js.git"
   },
-  "dependencies": {
-    "depd": "1.1.0"
-  },
   "devDependencies": {
-    "mocha": "latest",
+    "babel-cli": "^6.5.1",
+    "babel-plugin-add-module-exports": "^0.1.2",
+    "babel-plugin-transform-node-env-inline": "^6.5.0",
+    "babel-preset-es2015": "^6.5.0",
+    "babel-preset-es2015-rollup": "^1.1.1",
+    "coveralls": "latest",
+    "eslint": "^1.10.3",
+    "eslint-config-airbnb": "^5.0.0",
+    "glob": "^7.0.0",
     "istanbul": "latest",
-    "jshint": "latest",
-    "uglify-js": "latest",
-    "coveralls": "latest"
+    "mocha": "latest",
+    "rollup": "^0.25.3",
+    "rollup-plugin-babel": "^2.3.9",
+    "uglify-js": "latest"
   },
   "scripts": {
+    "lint": "eslint src",
+    "clean:node": "rm -rf index.js util lib",
+    "clean:browser": "rm -rf validator*.js",
+    "clean": "npm run clean:node && npm run clean:browser",
+    "minify": "uglifyjs validator.js -o validator.min.js  --compress --mangle --comments",
+    "build:browser": "NODE_ENV=browser babel-node build-browser && uglifyjs validator.js -o validator.js --beautify -c dead_code --comments && npm run minify",
+    "build:node": "NODE_ENV=node babel src -d . --presets es2015 --plugins babel-plugin-add-module-exports,transform-node-env-inline",
+    "build": "npm run build:browser && npm run build:node",
+    "pretest": "npm run lint && npm run build",
     "test": "node ./node_modules/.bin/_mocha --reporter spec",
     "coveralls": "istanbul cover _mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | coveralls && rm -rf ./coverage"
   },
-  "engines": { "node" : ">= 0.10" },
-  "license": "MIT"
+  "engines": {
+    "node": ">= 0.10"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "depd": "^1.1.0"
```

Comment:
Also keep this as it was. The dep will be dropped soon so there's no need to change it from `1.1.0` to `^1.1.0`.